packages/nuxt/src/components/runtime/server-component.ts

```diff
@@ -18,7 +18,7 @@ const SLOT_FALLBACK_RE = /<div nuxt-slot-fallback-start="([^"]*)"[^>]*><\/div>((
 const SSR_UID_RE = /nuxt-ssr-component-uid="([^"]*)"/
 
 let id = 0
-const getId = process.client ? () => (id++).toString() : randomUUID
+const getId = process.client ? () => (id--).toString() : randomUUID
```

Comment:
what about a prefix, like `'s-' + id`, which would also coerce the id to string

Reply:
sure !